debugger/src/utils.py

```diff
@@ -45,16 +45,16 @@ def get_subprocess_output(proc: subprocess.Popen, timeout_duration: int):
         for fileno in events:
             if fileno == proc.stdout.fileno():
                 print(
-                    f"=== Read from gdb subprocess stdout fileno = {fileno}:")
+                    f"˅˅˅˅˅ Read from gdb subprocess stdout fileno = {fileno}:")
```

Comment:
Why are we using ˅, shouldn't we avoid using not ascii chars?

Reply:
Good point, Will use `V` instead